src/variety/vslither.js

```diff
@@ -0,0 +1,420 @@
+		},
+		getTrial: function() {
+			return this.piece.trial;
+		}
+	},
+	Board: {
+		hasdots: 2,
+		hasborder: 2,
+		borderAsLine: true
+	},
+
+	LineGraph: {
+		enabled: true
+	},
+
+	//---------------------------------------------------------
+	// 画像表示系
+	Graphic: {
+		irowake: true,
+		bgcellcolor_func: "qsub2",
+		numbercolor_func: "qnum",
+		margin: 0.5,
+
+		paint: function() {
+			this.drawBGCells();
+			this.drawLines();
+			this.drawBaseMarks();
+			this.drawQuesNumbers();
+			this.drawPekes();
+			this.drawDots();
+			this.drawTarget();
+		},
+
+		repaintParts: function(blist) {
+			this.range.crosses = blist.crossinside();
+			this.drawBaseMarks();
+		},
+
+		drawDots: function() {
+			var g = this.vinc("dot", "auto");
+
+			g.lineWidth = (1 + this.cw / 40) | 0;
+			var d = this.range;
+			var size = this.cw * 0.15;
+			if (size < 3) {
+				size = 3;
+			}
+			var dlist = this.board.dotinside(d.x1, d.y1, d.x2, d.y2);
+			for (var i = 0; i < dlist.length; i++) {
+				var dot = dlist[i],
+					bx = dot.bx,
+					by = dot.by,
+					px = bx * this.bw,
+					py = by * this.bh;
+
+				g.vid = "s_dot_" + dot.id;
+				var outline = this.getDotOutlineColor(dot);
+				var color = this.getDotFillColor(dot);
+				if (dot.getDot() === 1) {
+					g.strokeStyle = outline;
+					g.fillStyle = color;
+					g.shapeCircle(px, py, this.cw * this.getDotRadius(dot));
+				} else if (dot.getDot() === 2) {
+					g.beginPath();
+					g.moveTo(px - size, py - size);
+					g.lineTo(px + size, py + size);
+					g.moveTo(px - size, py + size);
+					g.lineTo(px + size, py - size);
+					g.closePath();
+					g.stroke();
+				} else {
+					g.vhide();
+				}
+			}
+		},
+
+		getDotFillColor: function(dot) {
+			if (dot.getDot() === 1 || dot.getDot() === 2) {
+				return dot.getTrial() ? this.trialcolor : this.pekecolor;
+			}
+			return null;
+		},
+		getDotOutlineColor: function(dot) {
+			if (dot.getDot() === 1 || dot.getDot() === 2) {
+				return dot.getTrial() ? this.trialcolor : this.pekecolor;
+			}
+			return null;
+		},
+		getDotRadius: function(dot) {
+			return 0.1;
+		}
+	},
+
+	//---------------------------------------------------------
+	// URLエンコード/デコード処理
+	Encode: {
+		decodePzpr: function(type) {
+			this.decode4Cell();
+		},
+		encodePzpr: function(type) {
+			this.encode4Cell();
+		},
+
+		decodeKanpen: function() {
+			this.fio.decodeCellQnum_kanpen();
+		},
+		encodeKanpen: function() {
+			this.fio.encodeCellQnum_kanpen();
+		}
+	},
+	//---------------------------------------------------------
+	FileIO: {
+		decodeData: function() {
+			if (this.filever === 1) {
+				this.decodeCellQnum();
+				this.decodeCellQsub();
+				this.decodeBorderLine();
+			} else if (this.filever === 0) {
+				this.decodeCellQnum();
+				this.decodeBorderLine();
+			}
+		},
+		encodeData: function() {
+			this.filever = 1;
+			this.encodeCellQnum();
+			this.encodeCellQsub();
+			this.encodeBorderLine();
+		},
+		kanpenOpen: function() {
```

Comment:
You can remove everything within FileIO except for decodeData and encodeData. The other functions are for importing Slither Links from older tools (which don't support this newer type).

Reply:
Removed